jpeg/compress.go

```diff
@@ -44,6 +44,17 @@ static void encode_gray(j_compress_ptr cinfo, JSAMPROW pix, int stride) {
 	}
 }
 
+static void encode_rgba(j_compress_ptr cinfo, unsigned char pix[], int stride) {
+	JSAMPROW row_pointer[1];
+
+	int v;
+	for (v = 0; v < cinfo->image_height; v++) {
+		row_pointer[0] = &pix[v * stride];
+		jpeg_write_scanlines(cinfo, row_pointer, 1);
```

Comment:
use `v += jpeg_write_scanlines(cinfo, row_pointer, 1);` instead of increment in for statement.